src/components/CommentInfo/CommentInfo.jsx

```diff
@@ -1,3 +1,24 @@
-export const CommentInfo = () => (
-  <>Put the comment here</>
+export const CommentInfo = ({ comment }) => (
+  <>
+    <div className="CommentInfo">
+      <div className="CommentInfo__title">
+        <strong className="CommentInfo__name">
+          {comment.name}
+        </strong>
+
+        {' by '}
+
+        <a
+          className="CommentInfo__email"
+          href={comment.email}
```

Comment:
The href of the anchor tag is not properly formatted. It should be in the format of mailto:[email address] for it to function as a mail link.